pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixServerStarter.java

```diff
@@ -18,19 +18,54 @@
  */
 package org.apache.pinot.server.starter.helix;
 
-import com.google.common.base.Preconditions;
-import com.google.common.collect.ImmutableList;
+import static org.apache.pinot.common.utils.CommonConstants.Helix.CONFIG_OF_SERVER_FLAPPING_TIME_WINDOW_MS;
```

Comment:
import static org.apache.pinot.common.utils.CommonConstants.Helix.*;

Reply:
I'm using our pinot code style for eclipse which replaces imports `org.apache.pinot.common.utils.CommonConstants.Helix.*` to the explicit constant. Should we update the code style template? I typically prefer to stay consistent to what can be automated.